app/assets/stylesheets/cart.css.scss

```diff
@@ -88,3 +88,23 @@
   a:visited {color: #008000; text-decoration: underline; }
 }
 
+input[type="submit"].cart-item-quantity-button {
+  border : solid 0px #ffffff;
+  border-radius : .95em;
+  moz-border-radius : 3px;
+  -webkit-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
```

Comment:
Commas in function arguments should be followed by a single space<br>Property name should be immediately followed by a colon<br>`0px` should be written without units as `0`